### PR TITLE
feat(ng) Tooltip background

### LINKED ISSUE #1350: [DS to LF] Messages : Tooltip
Change the default background to 900

Capture d’écran 2021-05-26 à 17 34 36

### PR DESCRIPTION
**Pull request by RipripKipkip**
fixes #1350

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1380

File: packages/ng/root/src/style/theme/_tooltip.theme.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 $tooltip: (
-	background: _theme("commons.overlays.background", true),
-	color: _theme("commons.overlays.text", true),
+	background: _color("grey", 900, true),
```
Comment (vvalentin-lucca) [inline packages/ng/root/src/style/theme/_tooltip.theme.scss:2]:
```suggestion
	background-color: _color("grey", 900, true),
```

File: packages/ng/material/src/style/components/_tooltip.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .mat-tooltip {
 	border-radius: _theme("commons.border.radius");
-	background-color: _theme("commons.overlays.background");
-	color: white;
+	background-color: _color("grey", "900");
```
Comment (sanlucca) [inline packages/ng/material/src/style/components/_tooltip.scss:3]: It should be a variable, as some solutions uses a primary 900.

File: packages/ng/material/src/style/components/_tooltip.scss [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
+	color: _color("grey", "text");
```
Comment (sanlucca) [inline packages/ng/material/src/style/components/_tooltip.scss:4]: Can't we use the theming from tooltip.theme?

File: packages/ng/root/src/style/theme/_tooltip.theme.scss [hunk 4]
```diff
@@ -1,6 +1,6 @@
 $tooltip: (
-	background: _theme("commons.overlays.background", true),
-	color: _theme("commons.overlays.text", true),
+	background-color: _color("grey", 900, true),
```
Comment (sanlucca) [inline packages/ng/root/src/style/theme/_tooltip.theme.scss:2]: The key becomes "background-color", but I don't see any changes on how it used elsewhere. Is it normal ?

**Review by sanlucca — CHANGES_REQUESTED** [review]

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1380

**Review by sanlucca — APPROVED** [review]
It becomes a breaking change due to the variable name change. It needs to be documented as such

File: packages/ng/root/src/style/theme/_tooltip.theme.scss (hunk 4 above)
Comment (lucienbertin) [reply packages/ng/root/src/style/theme/_tooltip.theme.scss:2]: yeah, if its all the same to you i'd rather keep the same theme key so this PR isnt braking changes

**Pull request merged**